### ISSUE TITLE
regularization

### ISSUE THREAD

**Issue by jdvylder** [kkweon/UNet-in-Tensorflow#7]
Thanks for the project. 
When checking your code, I noticed you define l2 regularization, but don't explicitly add it to the loss function. According to https://bbabenko.github.io/weight-decay/ tensorflow does not do this automatically, so I guess your regularization is not used in this case.

kr

J.

**Comment by kkweon** [comment]
Ah you're right; That must be the reason why the new loss function didn't work well.  Thanks for reporting.
